docs/en/getting-started/install.md

```diff
@@ -19,7 +40,54 @@ $ grep -q sse4_2 /proc/cpuinfo && echo "SSE 4.2 supported" || echo "SSE 4.2 not
 
 To run ClickHouse on processors that do not support SSE 4.2 or have AArch64 or PowerPC64LE architecture, you should [build ClickHouse from sources](#from-sources) with proper configuration adjustments.
 
-## Available Installation Options {#available-installation-options}
```

Comment:
I think it's better to preserve links

Reply:
ok

Reply:
added back in, does not show in this diff, but if you view the whole file it is there.